thirdeye-server/src/main/java/ai/startree/thirdeye/core/AppAnalyticsService.java

```diff
@@ -78,6 +79,18 @@ protected Integer loadValue() {
         return uniqueMonitoredMetricsCount();
       }
     });
+    metricRegistry.register("anomalyCountTotal", new CachedGauge<Long>(1, TimeUnit.MINUTES) {
+      @Override
+      protected Long loadValue() {
+        return countTotal(null);
```

Comment:
this is bad by design: the one argument of the function is taking in is `null` as input.

Look at the other comments below. for cases like this, you can simply have an overloaded method. 
```
countTotal() { return countTotal(null); }
```

This is still frowned upon but not as much and depends on usecase. In this case, the DaoFilter input in itself seems not the best design choice.

Reply:
I have explained the reason for passing `DaoFilter` in above comment, let me know your thoughts on that. 
Was confused for introducing the overloaded method as right now its getting called from only one place so ended up not creating it. Let's sort the `DaoFilter` argument first and then I can make changes here based on that.